buildSrc/src/main/java/Koin.kt

```diff
@@ -0,0 +1,15 @@
+object Koin {
+    private const val VERSION = "3.2.0"
+
+    const val core = "io.insert-koin:koin-core:${VERSION}"
+    const val ktor = "io.insert-koin:koin-ktor:${VERSION}"
+
+    const val android = "io.insert-koin:koin-android:${VERSION}"
+    const val androidViewModel = "io.insert-koin:koin-androidx-viewmodel:2.2.3"
```

Comment:
Should this should be `${VERSION}` rather than hard-coded to `2.2.3` ?

Reply:
Koin Android View Model has a different version code than the rest of Koin